Makefile

```diff
@@ -6,7 +6,8 @@ MANIFEST := build/collections/ansible_collections/$(NAMESPACE)/$(NAME)/MANIFEST.
 
 ROLES := $(wildcard roles/*)
 PLUGIN_TYPES := $(filter-out __%,$(notdir $(wildcard plugins/*)))
-METADATA := galaxy.yml LICENSE README.md meta/runtime.yml requirements.txt changelogs/changelog.yaml CHANGELOG.rst bindep.txt
+RUNTIME_YML := meta/runtime.yml
```

Comment:
Is this file ever supposed to live somwhere else?

Reply:
no, I just wanted smth to refer to in multiple places